src/utils/transactions.ts

```diff
@@ -99,35 +99,27 @@ export const getDirection = (tx: Transaction, address: AddressHash): Transaction
   return amount && amountIsBigInt && amount < 0 ? 'out' : 'in'
 }
 
-export const calculateTotalTxOutput = (tx: Transaction) => {
-  const total = (tx.outputs ?? []).reduce((acc, o) => acc + BigInt(o?.attoAlphAmount ?? 0), BigInt(0))
-
-  // We must do best effort to determine the change addresses. Usually they are the last TXs.
-  const inputs = uniq((tx.inputs ?? []).map((i) => i.address))
-  let change = BigInt(0)
-
-  for (let index = (tx.outputs ?? []).length - 1; index > 0; index--) {
-    const output = (tx.outputs ?? [])[index]
-    const inputIndex = inputs.indexOf(output.address)
-    if (inputs[inputIndex]) {
-      change += BigInt(output.attoAlphAmount)
-      inputs.splice(inputIndex, 1)
-    }
-    if (inputs.length == 0) return total - change
-  }
```

Comment:
I find this calculation very complex to understand and reason about. I know that my solution might be slightly more computationally heavy, but is very simple.

Reply:
It is an inherently complex calculation. We will need tests to make sure your alternative is equivalent